src/System.Net.Http/tests/UnitTests/HttpSystemProxyTest.cs

```diff
@@ -80,9 +80,11 @@ public void HttpProxy_SystemProxy_Loaded(string rawProxyString)
 
                 Assert.True(HttpSystemProxy.TryCreate(out p));
                 Assert.NotNull(p);
+                Assert.Equal(expectedString, p.GetProxy(new Uri(fooHttp)).ToString());
+                Assert.Equal(expectedString, p.GetProxy(new Uri(fooHttps)).ToString());
```

Comment:
Why is "fooHttp" and "fooHttps" being used here?  Shouldn't it be "expectedString" ?

Reply:
p.GetProxy() expects destination URI to pass in and returns proxyURI. 
It does not really mater here but we used the foo* as test target elsewhere so it does not mix with proxy values.

Reply:
Ok.  Thx.